### PR TITLE
cfp2017 landing page

### PR DESCRIPTION
**Pull request by mpetitdant**

### CODE REVIEW

File: client-ng1/src/app/js/routes.js [hunk 1]
```diff
@@ -605,6 +605,10 @@
         .resolve({account: getAccount})
         .build())
 
+      .state('cfp2017', new State(USER_ROLES, 'cfp2017', 'js/cfp2017/cfp2017.html')
+        .controller('CfpCtrl2017')
+        .build())
+
```
Comment (javamind) [inline client-ng1/src/app/js/routes.js]: I think that we don't need a new page for that. 
We can add a section on the homepage no ? 
see `client-ng1/src/app/js/home/cf.directive.html`

File: client-ng1/src/app/js/cfp2017/cfp2017.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    <p class="cesar-text" translate="view.cfp.home.explanation"></p>
+
+    <div class="cesar-text-right">
+        <a href="https://mix-it.cfp.io" class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent" translate="view.cfp2017.home.submit"></a>
+    </div>
+
+    <p>&nbsp;</p>
+</div>
+
```
Comment (javamind) [inline client-ng1/src/app/js/cfp2017/cfp2017.html]: Should be moved in ``client-ng1/src/app/js/home/cf.directive.html`

File: client-ng1/src/app/js/routes.js (hunk 1 above)
Comment (javamind) [inline client-ng1/src/app/js/routes.js]: @clacote the real link is https://mix-it.cfp.io

Comment (clacote) [inline client-ng1/src/app/js/routes.js]: @javamind you're right.

**Comment by mpetitdant** [comment]
@javamind If ok can you handle the deployment ?

**Pull request merged**